### ISSUE TITLE
VO does not end reading when exiting any menu

### ISSUE THREAD

**Issue by brooklynlash** [phetsims/gravity-force-lab-basics#307]
**Test device**
MacBook Pro

**Operating System**
Mac 10.15.7

**Browser**
Chrome

**Problem description**
When reading a list on any menu (keyboard shortcuts, about menu, etc.) and exiting while the voice is still talking, the voice will not stop. The only way that the voice will stop is if you click something else to override it or change browser tabs. 

**Visuals**
Let me know if you need any and I can supply them. 

Troubleshooting information:
!!!!! DO NOT EDIT !!!!!
Name: ‪Gravity Force Lab: Basics‬
URL: https://phet-dev.colorado.edu/html/gravity-force-lab-basics/1.1.0-rc.2/phet/gravity-force-lab-basics_all_phet.html
Version: 1.1.0-rc.2 2021-08-30 16:27:03 UTC
Features missing: applicationcache, applicationcache, touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.114 Safari/537.36
Language: en-US
Window: 1680x868
Pixel Ratio: 2/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 15 uniform: 1024
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 80)
Max viewport: 16384x16384
OES_texture_float: true
Dependencies JSON: {}

**Comment by jessegreenberg** [comment]
Thanks @brooklynlash. I expected this case to be handled, checking to see why this is happening.
